src/material/schematics/ng-update/test-cases/v12/misc/theming-api-v12.spec.ts

```diff
@@ -734,4 +734,26 @@ describe('v12 theming API migration', () => {
       `@include mat.core();`,
     ]);
   });
+
+  it('should not add duplicate @use statements', async () => {
+    const originalContent = [
+      `@use '~@angular/material' as mat;`,
+      `@import '~@angular/material/theming';`,
+      `$something: mat.$red-palette;`,
+      `$another: $mat-pink;`,
+    ].join('\n');
+
+    const migratedContent = migrateFileContent(
```

Comment:
This should invoke the migration through the schematics API like the other tests, rather than calling `migrateFileContent`.

Reply:
Done